docs/githubteacher.md

```diff
@@ -0,0 +1,7 @@
+# GitHub Teacher
+
+**Name:** GitHub Teacher
+
+**Occupation:** Teacher
+
+**Hometown:** San Francisco
```

Comment:
There are a lot of San Franciscos in the :earth_africa:.  Maybe we should be more specific here.